src/main/java/com/zaurtregulov/spring/test/SimpleCalculator.java

```diff
@@ -0,0 +1,23 @@
+package com.zaurtregulov.spring.test;
+
+public class SimpleCalculator {
+
+    public double sumTwoNumbers(double a, double b) {
+        return a + b;
+    }
+
+    public double minusOneNumberFromAnother(double a, double b) {
+        return a - b;
+    }
+
+    public double divideOneNumberByAnother(double a, double b) {
+        if (b == 0)
+            return -1;
```

Comment:
you should throw exception when `b` is 0

Reply:
done